packages/apps/demo/src/lib/routes.js

```diff
@@ -0,0 +1,1826 @@
+/**
+ * Copyright 2023 Comcast Cable Communications Management, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Browse, Video } from '../pages';
+
+export default {
+  root: 'browse',
+  routes: [
+    {
+      path: 'browse',
+      component: Browse,
+      widgets: ['ThemeSwitcher'],
+      async on(page) {
+        try {
+          // Mock API Response
+          const data =
```

Comment:
can this `data` variable get broken out into a separate data file and imported here? It makes this routes file a little hard to parse.

Reply:
Moved to the static folder and making a request for JSON now